designs/UI/all_reviews.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>WeConnect</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" 
+    integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" 
```

Comment:
What do you think about placing the Js files just before closing </body>

Reply:
Good catch thanks. I remember reading somewhere that pages render better/faster if scripts are placed at the bottom of the page before closing. Is this the reason you for your recommend?

Reply:
@jm-wanja Done. That was a big ommission on my part.my part. Thanks

Reply:
@cosmas28 
Good catch thanks. I remember reading somewhere that pages render better/faster if scripts are placed at the bottom of the page before closing. Is this the reason you for your recommend